Services/Administration/OJS.Services.Administration.Models/ProblemResources/ProblemResourceAdministrationModel.cs

```diff
@@ -47,6 +48,19 @@ public void RegisterMappings(IProfileExpression configuration)
             .ForMember(pr => pr.ModifiedOn, opt
                 => opt.Ignore())
             .ForMember(pr => pr.Problem, opt
-                => opt.Ignore());
+                => opt.Ignore())
+            .AfterMap((pram, pr) =>
```

Comment:
`pram` and `pr` are a little bit unclear. Maybe `source` and `destination`?

Reply:
Done.